test/bigtest/tests/job-profiles/jobProfileDetails-test.js

```diff
@@ -27,6 +33,46 @@ describe('Job profile details: default job profile', () => {
     expect(jobProfileDetails.isPresent).to.be.true;
   });
 
+  describe('navigating to deleting confirmation modal', () => {
+    const callout = new CalloutInteractor();
+
+    beforeEach(async () => {
+      await jobProfileDetails.actionMenu.click();
+      await jobProfileDetails.actionMenu.deleteProfileButton.click();
+      await wait();
+    });
+
+    describe('deleting job profile successfully', () => {
+      beforeEach(async function () {
```

Comment:
We usually use `function` in cases when we need access to `this`, otherwise arrow functions.
```suggestion
      beforeEach(async () => {
```